environment.yml

```diff
@@ -1,30 +1,32 @@
 name: notebooks_env
 
-
 channels:
   - astropy
   - http://ssb.stsci.edu/astroconda
   - defaults
 
 dependencies:
+  - python=3.7
+  - ca-certificates=2019.8.28
+  - pip
+  - virtualenv=16.0.0
   - numpy=1.15.4
-  - astropy
-  - scipy
-  - astroquery
-  - matplotlib
-  - jupyter
-  - pandas
-  - drizzlepac
-  - crds
-  - ccdproc
-  - acstools
-  - photutils>=0.6
-  - regions
-  - pyregion
-  - webbpsf>=0.8
-  - scikit-learn
-
+  - astropy=3.0.4
```

Comment:
Why the shift to so many things in pip instead of conda?  One of the intentions here is to make sure the *astroconda-installed* version works as intended... `pip` is meant for only if something is not in `astroconda`.

Reply:
Ah, I think the answer is for performance. Possibly a good idea then, but I think this should be done as a separate *independent* PR because it has other consequences.  So I'd say just don't make any changes to the environment beyond what might be needed to support the new CCI machinery, and move these changes to a separate PR